static/app/components/modals/savedSearchModal/editSavedSearchModal.spec.tsx

```diff
@@ -27,6 +27,10 @@ describe('EditSavedSearchModal', function () {
       method: 'POST',
       body: [],
     });
+    MockApiClient.addMockResponse({
```

Comment:
Ignore all of `MockApiClient.addMockResponse({...})` additions made in this PR - [this prevents CI from failing](https://develop.sentry.dev/development/testing/#api-fixtures)